README.md

```diff
@@ -67,7 +67,7 @@ as will a non 200 type response.
 
 `:id` should be the unique Webchat room identifier
 
-This URL is used by the Webchat Component to open the correct Webchat instance.
+This URL is used by the Webchat Component to open the correct Webchat instance. The Webchat component should be able to call `open` on this endpoint and open a Webchat instance in a new window.
```

Comment:
Might be worth saying 'window.open' to be super clear what this means.

Reply:
@nickcolley Amended and pushed.